index.html

```diff
@@ -216,9 +213,11 @@ <h1>
   console.log(scratchblocks.stringify(doc));
   var docView3 = scratchblocks.newView(doc, {
     style: "scratch3",
+    scale: 0.675
```

Comment:
Neat!

Reply:
Actually this is the reason for this PR, because even Scratch itself is scaled.